.clang-tidy

```diff
@@ -47,11 +51,15 @@ CheckOptions:
     value:           '_'
   - key:             readability-identifier-naming.PrivateMemberSuffix
     value:           '_'
-  # const static or global variables are UPPER_CASE
+  # const static, global or member variables or are UPPER_CASE
   - key:             readability-identifier-naming.EnumConstantCase
     value:           UPPER_CASE
+  - key:             readability-identifier-naming.StaticVariableCasePrefix
+    value:           's_'
```

Comment:
What about the case where the static variable is mutable and not a constant?

Reply:
So far, we just had `lower_case` names for them.

Reply:
Our conclusion in the maintainer meeting was that we'd like to mark mutable static variables due to their impact on thread safety. Having a unique naming rule will give them more visibility for reviewers and developers. However, there was no strong preference for a specific naming rule. What would be your reasoning against a unique naming rule and if you don't have any objections, which naming convention would you prefere?

Reply:
If you want to mark mutable static variables, using a prefix `s_` is probably the way to go. Note that this will require many code changes, which all should be part of this PR.

Reply:
I hope that, generally, we don't use mutable static variables except in some generic code for edge cases. I don't know if that is true though :/